.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+[*]
+indent_style=tab
+indent_size=tab
```

Comment:
It's better to use the same config everywhere, probably copy this one: https://github.com/paritytech/eslint-config-parity/blob/master/.editorconfig